features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/modules/statistics/usage.jag

```diff
@@ -192,11 +199,11 @@ function getPerAppSubscribers(subscriberName,from,to, server) {
 }
 
 function isDataPublishingEnabled(providerName, server) {
-    
+
     var usage = [];
     var usageout = [];
     try {
-        usage = apiStore.isDataPublishingEnabled();
+        usage = usage = org.wso2.carbon.apimgt.usage.client.UsageClient.isDataPublishingEnabled();
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea